pkg/config/config.go

```diff
@@ -213,6 +213,11 @@ type Sources struct {
 	Plugins     Plugins          `koanf:",remain"`
 }
 
+// GetPlugins returns Sources.Plugins
```

Comment:
Please add a period for each new comment 👍 
```suggestion
// GetPlugins returns Sources.Plugins.
```